dime-coding-standards/checklists/Stata Code Review Checklist.md

```diff
@@ -0,0 +1,108 @@
+
+# Stata Peer Code Review Checklist
+
+Use this checklist to improve your own Stata coding practice or to guide your peer code reviews.
+
+
+## Main Do-file Setup
+    
+- [ ] Includes `clear all` and `set more off` at the start.
+- [ ] Sets core configurations (`version`, `matsize`, `varabbrev`) directly or via a wrapper (e.g., `ieboilstart`)
+- [ ] Script runs from start to end after changing directory paths in one place only.
+- [ ] Uses only **relative paths** (no `C:/...`) 
+- [ ] Uses forward slashes in file path for OS compatibility
+- [ ] Installs required packages or includes an `ado` folder with dependencies.
+- [ ] Sets a **random seed** for reproducibility.
+- [ ] Uses `run` or `do` to call modular scripts for cleaning, construction, etc.
```

Comment:
What would the alternative be? Does this relate to not manually running several scripts in order? If it is this, I think this is more to the point:

> The main-do file runs all code files (using `run` or `do` to files) without any need to manually run files in a certain order.